app/components/appMainModule/localePageModule/localePageRoutes.js

```diff
@@ -4,29 +4,64 @@ function localePageRoutes($stateProvider) {
 
   var localePage = {
     name: 'appMain.localePage',
-    url: '^/:localeType/:localeId/:localeName',   // The ^ character makes this url override the parent url
-    // template: '<{{ctrl.localeType}}-page><' + '/' + '{{ctrl.localeType}}' + '-page>',
-    // template: '<div ui-view="{{ctrl.localeType}}"></div>',
-    template: '<locale-page class="page-fade" locale-name="{{localeCtrl.localeName}}"></locale-page>',
+    url: '^/:localeType/:localeId/:localeName',
+    abstract: true,
+    template: '<locale-page class="page-fade" locale-name="{{ $ctrl.localeName }}"></locale-page>',
     controller: ['$rootScope', '$scope', '$stateParams', function($rootScope, $scope, $stateParams) {
-      var localeCtrl = this;
-      localeCtrl.localeName = $stateParams.localeName;
-      localeCtrl.localeType = $stateParams.localeType;
+      var ctrl = this;
+      ctrl.localeName = $stateParams.localeName;
+    }],
+    controllerAs: '$ctrl',
+    resolve: {
+      ballot: ['$stateParams', 'static_api', function($stateParams, static_api) {
+        return static_api.locality.current_ballot({locality_id: $stateParams.localeId});
+      }],
+      locality: ['$stateParams', 'static_api', function($stateParams, static_api) {
+        return static_api.locality.get({locality_id: $stateParams.localeId});
+      }]
+    }
+  };
 
+  var localeIndexPage = {
+    name: 'appMain.localePage.index',
+    url: '',
+    template: 'Click on a ballot item to the left to see campaign finance data in {{ $ctrl.locality.name  || "your city" }}.', // Placeholder for now
```

Comment:
Derp, we're not binding locality to the the controller, so this isn't going to render properly.